### PR TITLE
Add mozorg/about/governance/policies/participation.ftl

### PR DESCRIPTION
**Pull request by amychurchwell**
Ref: https://github.com/mozilla/bedrock/pull/9703

### CODE REVIEW

**Comment by amychurchwell** [comment]
@peiying2 may I have your review for en before adding all the other locales?

File: en/mozorg/about/governance/policies/participation.ftl [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+participation-consequences-of-unacceptable = Consequences of Unacceptable Behavior
+participation-bad-behavior-from = Bad behavior from any Mozillian, including those with decision-making authority, will not be tolerated. Intentional efforts to exclude people (except as part of a consequence of the guidelines or other official action) from { -brand-name-mozilla } activities are not acceptable and will be dealt with appropriately.
+participation-reports-of-harassmentdiscrimination = Reports of harassment/discrimination will be promptly and thoroughly investigated by the people responsible for the safety of the space, event or activity. Appropriate measures will be taken to address the situation.
+participation-anyone-asked-to-stop = Anyone asked to stop unacceptable behavior is expected to comply immediately. Violation of these guidelines can result in you being ask to leave an event or online space, either temporarily or for the duration of the event, or being banned from participation in spaces, or future events and activities in perpetuity.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/participation.ftl:85]: - `... being ask ...` should be `... being asked ...`
- `... you being ask ...` - `you` here seems strange. It started as `Anyone asked to stop...`. Need to use a different pronoun.

File: en/mozorg/about/governance/policies/participation.ftl [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+participation-in-addition-any-participants = In addition, any participants who abuse the reporting process will be considered to be in violation of these guidelines and subject to the same consequences. False reporting, especially to retaliate or exclude, will not be accepted or tolerated.
+participation-reporting = Reporting
+participation-if-you-believe-youre = If you believe you’re experiencing unacceptable behavior that will not be tolerated as outlined above, <a href="{ $hotline }">please use our hotline to report</a>. Reports go directly to { -brand-name-mozilla }’s Employment Counsel and HR People Partners and are triaged by the Community Participation Guidelines Response Lead.
+participation-after-receiving-a = After receiving a concise description of your situation, they will review and determine next steps. In addition to conducting any investigation, they can provide a range of resources, from a private consultation to other community resources. They will involve other colleagues or outside specialists (such as legal counsel), as needed to appropriately address each situation.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/participation.ftl:90]: Add `the` so it reads `... determine the next steps.`

File: en/mozorg/about/governance/policies/participation.ftl [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+participation-mozilla-spaces = { -brand-name-mozilla } Spaces
+participation-each-physical-or-virtual = Each physical or virtual { -brand-name-mozilla } space shall have a designated contact.
+participation-mozilla-events = { -brand-name-mozilla } Events
+participation-all-mozilla-events = All { -brand-name-mozilla } events will have designated a specific safety guideline with emergency and anti-abuse contacts at the event as well as online. These contacts will be posted prominently throughout the event, and in print and online materials. Event leaders are requested to speak at the event about the guidelines and to ask participants to review and agree to them when they sign up for the event.
+participation-reports-will-receive = Reports will receive an email notice of receipt. Once an incident has been investigated and a decision has been communicated to the relevant parties, all have the opportunity to appeal this decision by sending an email to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>.
+participation-ask-questions = Ask questions
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/participation.ftl:100]: Title case to stay consistent with other titles: `Ask Questions`

File: en/mozorg/about/governance/policies/participation.ftl [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+participation-everyone-is-encouraged = Everyone is encouraged to ask questions about these guidelines. If you are organizing an event or activity, reach out for tips building inclusion for your event, activity or space. Your input is welcome and you will always get a response within 24 hours (or on the next weekday, if it is the weekend) if you reach out to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>. Please <a href="{ $changelog }">review this change log</a> for updates to this document.
+participation-license-and-attribution = License and attribution
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/participation.ftl:102]: Title case to stay consistent with other titles: `License and attribution`

File: en/mozorg/about/governance/policies/participation.ftl [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+participation-this-set-of-guidelines = This set of guidelines is distributed under a <a href="{ $license }" rel="license">{ -brand-name-creative-commons } Attribution-ShareAlike license</a>.
+participation-these-guidelines-have = These guidelines have been adapted with modifications from { -brand-name-mozilla }’s original Community Participation Guidelines, the <a href="{ $ubuntu_coc }">Ubuntu Code of Conduct</a>, { -brand-name-mozilla }’s <a href="{ $viewsource_coc }">View Source Conference Code of Conduct</a>, and the <a href="{ $rustlang_coc }">{ -brand-name-rust } Language Code of Conduct</a>, which are based on Stumptown Syndicate’s <a href="{ $citizen_coc }">Citizen Code of Conduct</a>. Additional text from the <a href="{ $lgbtqtech_coc }">LGBTQ in Technology Code of Conduct</a> and the <a href="{ $wiscon_coc }">WisCon code of conduct</a>. This document and all associated processes are only possible with the hard work of many, many Mozillians.
+participation-modifications-to-these = Modifications to these guidelines
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/participation.ftl:105]: Title case to stay consistent with other titles: `Modifications to these guidelines`

File: en/mozorg/about/governance/policies/participation.ftl [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+participation-reports-will-receive = Reports will receive an email notice of receipt. Once an incident has been investigated and a decision has been communicated to the relevant parties, all have the opportunity to appeal this decision by sending an email to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>.
+participation-ask-questions = Ask questions
+participation-everyone-is-encouraged = Everyone is encouraged to ask questions about these guidelines. If you are organizing an event or activity, reach out for tips building inclusion for your event, activity or space. Your input is welcome and you will always get a response within 24 hours (or on the next weekday, if it is the weekend) if you reach out to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>. Please <a href="{ $changelog }">review this change log</a> for updates to this document.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/participation.ftl:101]: A word is missing after `tips`: `... reach out for tips building inclusion for your event ...`

**Review by peiying2 — CHANGES_REQUESTED** [review]
@amychurchwell Other than what's called out, the URLs are not called out separately in this file either, like the reporting PR.

File: en/mozorg/about/governance/policies/participation.ftl [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+participation-disruptive-behavior = Disruptive Behavior
+participation-sustained-disruption = Sustained disruption of events, forums, or meetings, including talks and presentations, will not be tolerated. This includes:
+participation-talking-over-or-heckling = ‘Talking over’ or ‘heckling’ speakers.
+participation-drinking-alcohol-to = Drinking alcohol to excess or using recreational drugs to excess, or pushing others to do so.
+participation-making-derogatory = Making derogatory comments about those who abstain from alcohol or other substances, pushing people to drink, talking about their abstinence or preferences to others, or pressuring them to drink - physically or through jeering.
+participation-otherwise-influencing = Otherwise influencing crowd actions that cause hostility in the session.
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply en/mozorg/about/governance/policies/participation.ftl:79]: It reads to me like "any other kind of influencing crowd actions" I am not sure what the best grammar would be.

File: en/mozorg/about/governance/policies/participation.ftl [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+participation-unwelcome-sexual-attention = Unwelcome Sexual Attention or Physical Contact
+participation-unwelcome-sexual-attention-or = Unwelcome sexual attention or unwelcome physical contact is not acceptable. This includes sexualized comments, jokes or imagery in interactions, communications or presentation materials, as well as inappropriate touching, groping, or sexual advances. This includes touching a person without permission, including sensitive areas such as their hair, pregnant stomach, mobility device (wheelchair, scooter, etc) or tattoos. This also includes physically blocking or intimidating another person. Physical contact or simulated physical contact (such as emojis like “kiss”) without affirmative consent is not acceptable. This includes sharing or distribution of sexualized images or text.
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply en/mozorg/about/governance/policies/participation.ftl:73]: What about:

> Unwelcome sexual attention or unwelcome physical contact is not acceptable. This includes sexualized comments, jokes or imagery in interactions, communications or presentation materials, as well as inappropriate touching, groping, or sexual advances. Additionally, touching a person without permission, including sensitive areas such as their hair, pregnant stomach, mobility device (wheelchair, scooter, etc) or tattoos is unacceptable. This includes physically blocking or intimidating another person. Physical contact or simulated physical contact (such as emojis like “kiss”) without affirmative consent is not acceptable. The sharing or distribution of sexualized images or text is unacceptable.

File: en/mozorg/about/governance/policies/participation.ftl (hunk 7 above)
[the remark this answers is not in the document]
Comment (peiying2) [reply en/mozorg/about/governance/policies/participation.ftl:79]: Let's use your suggested text instead.

**Review by peiying2 — APPROVED** [review]
Looks good. Thanks for reworking the long sentence. Please add the localized files.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**